packages/flutter/lib/src/material/switch.dart

```diff
@@ -423,6 +438,16 @@ class _RenderSwitch extends RenderToggleable {
     markNeedsPaint();
   }
 
+  DragStartBehavior get dragStartBehavior => _dragStartBehavior;
+  DragStartBehavior _dragStartBehavior;
```

Comment:
as with the cupertino case, you can skip the field here and just have the getter directly read _drag.dragStartBehavior